config/teleop.yaml

```diff
@@ -1,4 +1,6 @@
 teleop:
+  ra_joints: ["joint_a", "joint_b", "joint_c", "joint_d", "joint_e"]
```

Comment:
wait so its confirmed that we can get away with5  joints only?

Reply:
Nah this is just because these are the only 5 joints we are actually controlling and need feedback from since f has no encoder.